source/php/Component/Slider__item/slider__item.blade.php

```diff
@@ -21,7 +21,10 @@
                     "variant" => "h2",
                     "element" => "h2",
                     'classList' => [$baseClass.'__heading'],
-                    'autopromote' => true
+                    'autopromote' => true,
+                    'attributeList' => [
+                        'id' => $uid . '__heading',
```

Comment:
Use parameter id for the component. No need to use attributeList for native parameters.